guides/common/modules/proc_registering-a-host.adoc

```diff
@@ -27,59 +27,75 @@ endif::[]
 ifdef::katello,satellite,orcharhino[]
-. In the *Activation Keys* field, enter one or more activation keys to assign to hosts.
+. In the *Activation Keys* field, enter one or more activation keys to assign to your host.
 endif::[]
-. Optional: Select the *Insecure* option, if you want to make the first call insecure.
-During this first call, hosts download the CA file from {Project}.
-Hosts will use this CA file to connect to {Project} with all future calls making them secure.
+. Optional: If you want to use `wget` to register your host to {Project}, select the *Use wget* option.
+By default, {Project} generates a `curl` command.
+. Optional: If you want to make the first call insecure, select the *Insecure* option.
+During this first call, your host downloads the CA file from {Project}.
+Your host will use this CA file to connect to {Project} with all future calls making them secure.
 +
 {Team} recommends that you avoid insecure calls.
 +
-If an attacker, located in the network between {Project} and a host, fetches the CA file from the first insecure call, the attacker will be able to access the content of the API calls to and from the registered host and the JSON Web Tokens (JWT).
-Therefore, if you have chosen to deploy SSH keys during registration, the attacker will be able to access the host using the SSH key.
+If an attacker, located in the network between {Project} and your host, fetches the CA file from the first insecure call, the attacker will be able to access the content of the API calls to and from your host and the JSON Web Tokens (JWT).
+Therefore, if you have chosen to deploy SSH keys during registration, the attacker will be able to access your host using the SSH key.
 +
-Instead, you can manually copy and install the CA file on each host before registering the host.
+Instead, you can manually copy and install the CA file on each host before registration.
 +
 To do this, find where {Project} stores the CA file by navigating to *Administer* > *Settings* > *Authentication* and locating the value of the *SSL CA file* setting.
 +
-Copy the CA file to the `/etc/pki/ca-trust/source/anchors/` directory on hosts and enter the following commands:
+Copy the CA file to the `/etc/pki/ca-trust/source/anchors/` directory on your host and enter the following commands:
 +
 [options="nowrap", subs="+quotes,attributes"]
 ----
 # update-ca-trust enable
 # update-ca-trust
 ----
 +
-Then register the hosts with a secure `curl` command, such as:
+Then register your host with a secure `curl` command, such as:
 +
 [options="nowrap", subs="+quotes,attributes"]
 ----
 # curl -sS https://{foreman-example-com}/register ...
 ----
 +
-The following is an example of the `curl` command with the `--insecure` option:
+The following is an example of the insecure `curl` command with the `--insecure` option:
 +
 [options="nowrap", subs="+quotes,attributes"]
 ----
 # curl -sS --insecure https://{foreman-example-com}/register ...
 ----
```

Comment:
```suggestion
+
If your host does not trust the SSL cerfiticate of your {ProjectServer}, add the `--insecure` option to the registration command.
```

Reply:
Yeah, but the user doesn't add the option manually, they select the option in the GUI.

Reply:
Yes, my bad. Then it'd be "...., select the *Insecure* checkbox". @goarsna